### PR TITLE
Add filtering by customfield ranges

### PR DESCRIPTION
**Pull request by zymeth25**
I implemented my changes described in #324 

@likemusic please have a look and try it.

### CODE REVIEW

**Comment by likemusic** [comment]
@zymeth25 Thanx for implementing! I will try it in coming days.

File: include/lcp-utils.php [hunk 1]
```diff
@@ -37,4 +37,25 @@ public static function lcp_show_pagination($pagination){
...
+    switch ($type) {
+      case 'DATETIME':
+        $format = 'c';
+        break;
+      case 'DATE':
+        $format = 'Y-m-d';
+        break;
+      case 'TIME':
+        $format = 'H:i:s';
+    }
```
[the remark this answers is not in the document]
Comment (zymeth25) [reply include/lcp-utils.php]: DB specific, the formats listed in WP_Meta_Query documentation are just copied from MySQL docs.

MySQL accepts a "relaxed" (as they call it) syntax for Date and Time string literals, but I used the ones explicitly given in [the docs](https://dev.mysql.com/doc/refman/5.7/en/date-and-time-literals.html).

Also, as you see I only implemented formatting for those types that need it for an average user. When someone is using DECIMAL they probably know what they are doing and there is no need to help them 😉

[the remark this answers is not in the document]
Comment (picandocodigo) [reply include/lcp-utils.php]: Cool, good to know! Thanks :+1:

**Comment by zymeth25** [comment]
This is good to go too, works in my tests.

**Comment by zymeth25** [comment]
Closing this because I will be changing the implementation as described in #428.

**Pull request closed without merging**